wlms/ircbridge.go

```diff
@@ -6,8 +6,29 @@ import (
 	"strings"
 )
 
+// Structure with channels for communication between IRCBridger and the metaserver
```

Comment:
typo? IRCBridger. Should that not be IRCBridge everywhere?

Reply:
At least no typo of mine. For some reason the interface class of the IRC bridge is named that way. IRCBridger is the interface, IRCBridge the struct/class which implements it...